<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogInfoControlElement</class>
 <widget class="QDialog" name="DialogInfoControlElement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Элемент контроля</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Оценки студентов</string>
   </property>
   <widget class="QTableWidget" name="studentsTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="setMarkButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Поставить оценку</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="markSpinBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QComboBox" name="studentsComboBox">
    <property name="geometry">
     <rect>
      <x>328</x>
      <y>50</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
